packages/manager/modules/pci/src/components/pci-order-command-terraform/pci-order-command-terraform.controller.js

```diff
@@ -0,0 +1,39 @@
+export default class PciOrderCommandTerraformCtrl {
+  $onInit() {
+    this.identationString = '  ';
```

Comment:
**nitpick:** This property is only used in `getIdentationString`. So no need to attach it to controller object instead it can be defined in required function as a local variable.

Reply:
Fixed 22aadb289f45d162914c683a33a768cda86d51a1 :)